src/util/process.cpp

```diff
@@ -32,7 +32,7 @@ void process::run() {
         buffer<char*> pargs;
         for (auto arg : m_args) {
             std::cout << arg << std::endl;
```

Comment:
Github doesn't let me comment a few lines down on `pargs.data()[pargs.size()] = NULL;`, so I'll do it here.  The only reason this works is because buffer starts out with a capacity of 16, otherwise this might already be a write to unallocated memory.  I'd use `pargs.push_back(nullptr)` here.

Also `pargs.data()[0]` seems redundant, doesn't `pargs[0]` work as well?